### PR TITLE
Update dg setcrs

### PR DESCRIPTION
**Pull request by saiutkarsh33**
- Edited storage puml diagram and notes in DG
- Edited model notes in DG
- Edited user stories in DG (solves #240)

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -388,10 +389,17 @@ _{Explain here how the data archiving feature will be implemented}_
...
+| {{ threeStars }} | TA                  | know all the commands of TAPro via the help window                          | use it effectively                             |
+| {{ threeStars }} | TA                  | see all students in the contact book                                        | have an overview of all students               |
+| {{ threeStars }} | TA                  | edit a student's details                                                    | have the latest data                           |
+| {{ threeStars }} | TA                  | find a student by name                                                      | get a student's data easily                    |
+| {{ threeStars }} | TA                  | delete all students from the contact book                                   | reset my student's data quickly                |
+| {{ twoStars }}   | TA                  | retrieve command history                                                    | avoid retyping a command                       |
+| {{ twoStars }}   | TA                  | use TAB to autocomplete my input                                            | to save time                                   |
```
Comment (tituschewxj) [inline docs/DeveloperGuide.md:401]: We should not be too specific in user stories about how a feature is implemented
```suggestion
| {{ twoStars }}   | TA                  | to autocomplete my input                                            | to save time                                   |
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -388,10 +389,17 @@ _{Explain here how the data archiving feature will be implemented}_
...
 | {{ threeStars }} | TA                  | delete a student                                                            | remove a student if he or she leaves the class |
-| {{ threeStars }} | TA                  | know all the commands of TAPro                                              | use it effectively                             |
+| {{ threeStars }} | TA                  | know all the commands of TAPro via the help window                          | use it effectively                             |
+| {{ threeStars }} | TA                  | see all students in the contact book                                        | have an overview of all students               |
+| {{ threeStars }} | TA                  | edit a student's details                                                    | have the latest data                           |
+| {{ threeStars }} | TA                  | find a student by name                                                      | get a student's data easily                    |
+| {{ threeStars }} | TA                  | delete all students from the contact book                                   | reset my student's data quickly                |
```
Comment (tituschewxj) [inline docs/DeveloperGuide.md:399]: We should be more specific with this user story.
```suggestion
| {{ threeStars }} | TA                  | delete all students from a previous semester from the contact book                                   | clear my contacts quickly at the start of a semester                |
```

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -388,10 +389,17 @@ _{Explain here how the data archiving feature will be implemented}_
 | {{ threeStars }} | TA {{ whitespace }} | name/rename the CS course that I am tutoring this semester                  | keep track of the module I am teaching         |
 | {{ threeStars }} | TA                  | add a student to the CS course that I am tutoring that semester to my class | keep track of him or her                       |
 | {{ threeStars }} | TA                  | view all students from my class                                             | view details about all of them                 |
-| {{ threeStars }} | TA                  | mark attendance for a student in my class                                   | keep track of who's present                    |
-| {{ threeStars }} | TA                  | unmark attendance for a student in my class                                 | keep track of who is absent                    |
+| {{ threeStars }} | TA                  | mark attendance for a student in my class for a particular week             | keep track of who's present                    |
+| {{ threeStars }} | TA                  | unmark attendance for a student in my class for a particular week           | keep track of who is absent                    |
```
Comment (tituschewxj) [inline docs/DeveloperGuide.md:393]: Perhaps we can be more consistent with our phrasing:
```suggestion
| {{ threeStars }} | TA                  | mark attendance for a student in my class for a particular week             | keep track of who is present                    |
| {{ threeStars }} | TA                  | unmark attendance for a student in my class for a particular week           | keep track of who is absent                    |
```

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -388,10 +389,17 @@ _{Explain here how the data archiving feature will be implemented}_
 | {{ threeStars }} | TA {{ whitespace }} | name/rename the CS course that I am tutoring this semester                  | keep track of the module I am teaching         |
 | {{ threeStars }} | TA                  | add a student to the CS course that I am tutoring that semester to my class | keep track of him or her                       |
```
Comment (tituschewxj) [inline docs/DeveloperGuide.md:390]: The wording can be improved:
```suggestion
| {{ threeStars }} | TA                  | add a student to my class that I am tutoring this semester | keep track of him or her                       |
```

**Review by tituschewxj — CHANGES_REQUESTED** [review]
Thanks for updating this. I have a few nits.

---

Can we update the priorities high, medium, low to the following (don't remove the icon):
* High: (must have)
* Medium: (should have)
* Low: (nice to have)

This would be more in line with what we have in our user stories.

---

Additionally, can we add the user story for syntax highlighting: 
* As a TA, I want to read the output more quickly, so that I can be more efficient.

**Review by belligerentbeagle — APPROVED** [review]
LGTM besides Titus' comments, can merge once fixed and req for Titus' approval too!

**Review by tituschewxj — APPROVED** [review]
LGTM!

**Pull request merged**